README.md

```diff
@@ -0,0 +1,27 @@
+# recheck for web
+
+Replacing manual asserts: check everything at once.
+
+[![license](https://img.shields.io/badge/license-AGPL-brightgreen.svg)](https://github.com/retest/recheck-web/blob/master/LICENSE) [![PRs welcome](https://img.shields.io/badge/PRs-welcome-ff69b4.svg)](https://github.com/retest/recheck-web/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) [![code with hearth by retest](https://img.shields.io/badge/%3C%2F%3E%20with%20%E2%99%A5%20by-retest-C1D82F.svg)](https://github.com/retest)
+
+## Features
```

Comment:
Isn't a lot of this "future stuff"? For instance, there's no JAR or Maven dependency one can use right now. I wouldn't add features, documention, etc. right now. Why not wait until we have actually built it?